README.md

```diff
@@ -74,6 +74,7 @@ public async Task Example()
 | ✅      | [Providers](#providers) | Integrate with a commercial, open source, or in-house feature management tool.                                                     |
 | ✅      | [Targeting](#targeting) | Contextually-aware flag evaluation using [evaluation context](https://openfeature.dev/docs/reference/concepts/evaluation-context). |
 | ✅      | [Hooks](#hooks)         | Add functionality to various stages of the flag evaluation life-cycle.                                                             |
+| ✅      | [Tracking](#tracking)   | Associate user actions with feature flag evaluations.                                                                              |
```

Comment:
I couldn't suggest this so I just added it - hope you don't mind. This is useful here but it's also used to build [this](https://openfeature.dev/docs/reference/technologies/sdk-compatibility).